test/box/gh-6405-discarding-arguments-in-box-func-call.test.lua

```diff
@@ -0,0 +1,55 @@
+sum_args = function(...)                      \
+    local type_table = {}                     \
+    for i = 1, select('#', ...) do            \
+    type_table[i] = type(select(i, ...))      \
+    end                                       \
+    return type_table                         \
+end                                           \
```

Comment:
Can't see sum here :) 
I think `sum_args` is not appropriate name for this function.

Reply:
fixed